src/ui/Video/Video.stories.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Container } from 'semantic-ui-react';
+import NextCloudEmbed from '@eeacms/volto-nextcloud-video-block/NextCloud/players/NextCloudEmbed';
```

Comment:
@tarantilis I don't think this should be a hard dependency to `@eeacms/volto-nextcloud-video-block`